### PR TITLE
Add getting started code samples for MeiliSearch documentation

### PR DESCRIPTION
**Pull request by tpayet**
Add code samples for MeiliSearch getting started.

### CODE REVIEW

File: .code-samples.meilisearch.yaml [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+settings_guide_searchable_1: |-
+settings_guide_displayed_1: |-
+settings_guide_accept_new_fields_1: |-
+add_movies_json_1: |-
+documents_guide_add_movie_1: |-
+search_guide_1: |-
+search_guide_2: |-
+getting_started_create_index_md: |-
+  ```bash |-
+    $ bundle add meilisearch
```
Comment (bidoubiwa) [inline .code-samples.meilisearch.yaml:84]: bundle add or gem install?

Comment (tpayet) [reply .code-samples.meilisearch.yaml:84]: `bundle add` add gem to the Gemfile and run bundle install 😎

File: .code-samples.meilisearch.yaml [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+  [About this gem](https://www.github.com/meilisearch/meilisearch-ruby)
+getting_started_add_documents_md: |-
+  ```ruby
+  require 'json'
+
+  movies_json = File.read('movies.json')
+  movies = JSON.parse(movies_json)
+  index.add_documents(movies)
+  ```
+getting_started_search_md: |-
```
Comment (curquiza) [inline .code-samples.meilisearch.yaml:103]:
```suggestion

  [About this gem](https://www.github.com/meilisearch/meilisearch-ruby)
getting_started_search_md: |-
```

File: .code-samples.meilisearch.yaml [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+  ```ruby
+  index.search('botman')
+  ```
+faceted_search_update_settings_1: |-
```
Comment (curquiza) [inline .code-samples.meilisearch.yaml:107]:
```suggestion

  [About this gem](https://www.github.com/meilisearch/meilisearch-ruby)
faceted_search_update_settings_1: |-
```

File: .code-samples.meilisearch.yaml [hunk 4]
```diff
@@ -0,0 +1,112 @@
+# This code-sample file is used by the MeiliSearch documentation
+# Every examples written here will be automatically fetched by
+# the documentation on build
+# You can read more on https://github.com/meilisearch/documentation/tree/master/.vuepress/code-samples
+---
```
Comment (eskombro) [inline .code-samples.meilisearch.yaml:5]:
```suggestion
```

Comment (curquiza) [reply .code-samples.meilisearch.yaml:5]: I think we should put this kind of comment in code-sample file. Do you?

Comment (eskombro) [reply .code-samples.meilisearch.yaml:5]: Actually that's true. I didn't use it, but I think you are right!

Comment (curquiza) [reply .code-samples.meilisearch.yaml:5]: But there is a typo ^^

File: .code-samples.meilisearch.yaml [hunk 5]
```diff
@@ -0,0 +1,114 @@
+# This code-sample file is used by the MeiliSearch documentation
+# Every examples written here will be automatically fetched by
+# the documentation on build
+# You can read more on https://github.com/meilisearch/documentation/tree/master/.vuepress/code-samples
```
Comment (curquiza) [inline .code-samples.meilisearch.yaml:4]:
```suggestion
# This code-sample file is used by the MeiliSearch documentation
# Every example written here will be automatically fetched by
# the documentation on build
# You can read more on https://github.com/meilisearch/documentation/tree/master/.vuepress/code-samples
```

File: .code-samples.meilisearch.yaml (hunk 4 above)
Comment (eskombro) [reply .code-samples.meilisearch.yaml:5]: Hahaha now we have the typo in a loooot of repositories 😄

**Pull request merged**